_includes/tutorials/produce-consume-lang/scala/markup/dev/3_4-schema-serde.adoc

```diff
@@ -0,0 +1,21 @@
+The case class instances in our code will be serialized in an https://avro.apache.org/[Avro] format.
+To do so, we will need the appropriate serializers/deserializers.
+We add the following function to easily create a `Serializer[T]` where `T` in the type of the key or the value
```

Comment:
Missing the path and name for creating the `ScalaReflectionSerde.scala` file in the text.